examples/pytorch/image-classification/run_image_classification.py

```diff
@@ -20,6 +20,7 @@
 from dataclasses import dataclass, field
 from typing import Optional
 
+# Importing Evaluation metrics and image dataset transformation packages
```

Comment:
this comment is unnecessary. can you remove it? generally for simple imports comments are not used.